core/geometry/src/curve/internalContexts/geometry2d/ImplicitCurve2dLocationDetail.ts

```diff
@@ -0,0 +1,98 @@
+
+    /** point getter -- returns a CLONE of the point in the detail. */
+    public get point(): Point2d | undefined {
+        return this._point === undefined ? undefined : this._point.clone();
+    }
+    /** point setter -- savs a CLONE of the input point. */
+    public set point(pointToClone: Point2d) { this._point = pointToClone.clone(); }
+
+    /** _gradientVector getter -- returns a CLONE of the _gradientVector in the detail. */
+    public get gradientVector(): Vector2d | undefined {
+        return this._gradientVector === undefined ? undefined : this._gradientVector.clone();
+    }
+    /** _gradientVector setter -- savs a CLONE of the input _gradientVector. */
+    public set gradientVector(vectorToClone: Vector2d) { this._gradientVector = vectorToClone.clone(); }
+
+    /** tangentVector getter -- returns a CLONE of the tangentVector in the detail. */
+    public get tangentVector(): Vector2d | undefined {
+        return this._tangentVector === undefined ? undefined : this._tangentVector.clone();
+    }
+    /** tangentVector setter -- savs a CLONE of the input tangentVector. */
+    public set tangentVector(vectorToClone: Vector2d) { this._tangentVector = vectorToClone.clone(); }
+
+    /** angle getter -- returns a CLONE of the angle in the detail. */
+    public get angle(): Angle | undefined {
+        return this._angle === undefined ? undefined : this._angle.clone();
+    }
+    /** angle setter -- savs a CLONE of the input angle. */
+    public set angle(angleToClone: Angle) { this._angle = angleToClone.clone(); }
+
+    /** functionValue getter -- returns the stored function value. */
+    public get functionValue(): number | undefined {
+        return this._functionValue;
+    }
+    /** functionValue setter -- savs a the input value. */
+    public set functionValue(value: number | undefined) { this._functionValue = value; }
+
+    private constructor(curve: ImplicitCurve2d, point?: Point2d, angle?: Angle) {
+        this._point = (point === undefined) ? Point2d.createZero() : Point2d.createZero();
+        this._curve = curve;
+        this._angle = angle;
+    }
+
+    /**
+     * Create a detail with a curve but all other content undefined.
+     * @param curve curve reference.  This reference is copied into the detail -- not cloned.
+     * @returns 
+     */
+    public static createDetail(curve: ImplicitCurve2d, point : Point2d | undefined, angle : Angle | undefined) {
+        return new ImplicitCurve2dLocationDetail(curve, point, angle);
+    }
+    /** Evaluate the curve at given angle and aave the angle and point. */
+    public evaluateCurvePoint (angle: Angle){
+        this._angle = angle;
+        this._point = this._curve.radiansToPoint2d (angle.radians);
+    }
+    /** Evaluate the curve point, tangent, gradient, and function
+     */
+    public evaluatePointFunctionGradient (angle: Angle) : boolean{
+        this._angle = angle;
+        this._point = this._curve.radiansToPoint2d (angle.radians);
```

Comment:
Fix here and next lines
```suggestion
        this._point = this._curve.radiansToPoint2d(angle.radians);
```